README.md

```diff
@@ -258,21 +258,590 @@ As part of the deployment, you'll need to set up [git integration](https://rasa.
+
+- In your AWS Console, go to IAM dashboard to create a new set of API keys:
+
+  - Click on Users
+
+  - Click on Add user
+
+    - User name = findemo  *(The actual name is not important, we will never use this name directly)*
+
+      Choose "**programmatic access**." This allows you to use the aws cli to interact with AWS.
+
+    - Click on Next: Permissions
+
+      - Click on *Attach existing policies directly*
+
+        For IAM access, you can choose “**AdministratorAccess**”, or limit access to only what is needed by the CD pipeline.
+
+    - Click on Next: Tags
+
+    - Click on Next: Review
+
+    - Click on Create user
+
+    - Store in a safe location: `Access key ID` & `Secret access key`
+
+
+### SSH Key Pair
+
+To be able to SSH into the EC2 worker nodes of the EKS cluster, you need an SSH Key Pair
+
+- In your AWS Console, go to **EC2 > Key Pairs**, and create a Key Pair with the name `findemo`, and download the file `findemo.pem` which contains the private SSH key. 
+  **Note that the name `findemo` is important, since it is used by the CI/CD pipeline when the cluster is created.**
+
+### Local AWS CLI
+
+Before the CI/CD pipeline can run, you will use the AWS CLI locally to create some items, so also install & configure the CLI locally.
+
+#### Install AWS CLI v2
+
+See the [installation instructions](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+#### Configure your AWS CLI
+
+```bash
+# AWS CLI version 2.1.26 or later
+aws --version
+
+# Configure AWS CLI
+aws configure
+AWS Access Key ID [None]: -----          # See above: IAM User API Keys
+AWS Secret Access Key [None]: -------    # See above: IAM User API Keys
+Default region name [None]: us-west-2    # The CI/CD pipeline uses us-west-2 
+Default output format [None]: 
+
+# Check your configuration
+aws configure list [--profile profile-name]
+
+# verify it works
+aws s3 ls
+```
+
+### ECR repository & S3 bucket
+
+The CI pipeline creates two artifacts:
+
+- An action server docker **image**, which is pushed to an **AWS ECR** repository.
+- A trained rasa **model**, which is copied to an **AWS S3** bucket
+
+Run these commands to create the storage for these artifacts. These commands run the AWS CLI and create an ECR repository and an S3 bucket:
+
+```bash
+# create ECR repository with name `financial-demo`
+make aws-ecr-create-repository
+
+# create S3 bucket with name `rasa-financial-demo`
+make aws-s3-create-bucket
+```
+
+
+
+## EKS production cluster
+
+If the production cluster is already set up, you can skip this section.
+
+This section describes the initial deployment of the financial-demo bot on an EKS production cluster. 
+
+This initial deployment is done manually. After that, the deployment is maintained & upgraded automatically by the CI/CD pipeline.
+
+### Preparation
```

Comment:
Can you add a brief sentence or two here summarizing what's needed for the preparation step?

Reply:
The **Preparation** section starts on the next line, so I added a summary there. 
Let me know if that works.